tfhe/src/high_level_api/mod.rs

```diff
@@ -130,23 +130,23 @@ pub enum Device {
 
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
 pub enum FheTypes {
```

Comment:
this needs to be marked #[repr(i32)] and all the types need to have a value associated, then we'll check with BC it's fine with them